_sass/_base.scss

```diff
@@ -172,11 +172,10 @@ table {
 $blockquote-color: $primary-text-color-dark;
 
 blockquote {
-  @include type-setting(2);
+  @include type-setting(1);
```

Comment:
Updated 👍 

Related, I also requested some margin on the footer: https://github.com/radicalbarracuda/sf-feminist-book-club/pull/25